samples/Integration/CMake-fetch-content-app/README.md

```diff
@@ -0,0 +1,42 @@
+# Integrating the Azure SDK for C++ with CMake fetch content to your application
+
+This application shows how to integrate the Azure SDK for C++ in your application. It uses CMake fetch content functionality to clone the Azure SDK for C++ repo into your `build` directory. This approach is useful when using CMake to build your application.
+
+## Pre-requisites
+
+Install the [Azure SDK for C++ dependencies](https://github.com/Azure/azure-sdk-for-cpp/blob/master/CONTRIBUTING.md#third-party-dependencies).
+
+- CMake project (min version 3.13).
+- C++ version 14 or greater.
+
+## Build
+
+```bash
+#
+# Building the application.
+# Instructions from application root directory.
+#
+
+# Create build directory just the first time.
+mkdir build
+cd build
+
+# Generate and build
+# This code assumes that the SDK dependencies were installed with VCPKG
+cmake -DCMAKE_TOOLCHAIN_FILE=path/to/vcpkg/scripts/buildsystems/vcpkg.cmake ..
+cmake -build .
+```
+
+## Run application
+
+Review source code header for `environment variables` that must be set up before running the app.
```

Comment:
What happens if I run it with none of the environment variables set? Does it tell me which ones I need to set?

Reply:
naaa,  it crashes because when we call getenv() and it doesn't fin the variable, there's an exception.